spec/predicates/scai.md

```diff
@@ -1,149 +1,150 @@
 # Predicate type: Software Supply Chain Attribute Integrity (SCAI)
 
-Type URI: scai/attribute-report/v0.1
+Type URI: https://in-toto.io/scai/attribute-report
```

Comment:
Good question. I like the idea of adding /attestation to the URL (maybe even /attestation/predicate for predicates). I think this could be resolved in a single PR, if need be, though.

Reply:
Sounds good, I think these could be part of the not-entirely-but-almost-stable changes we discussed finishing pre 1.1.